extensions/classifiers/SVMPredictionService.js

```diff
@@ -95,7 +104,7 @@ oppia.factory('SVMPredictionService', ['$log', function($log) {
             sum += kvalues[sj + k] * coef2[sj + k];
           }
 
-          sum -= intercept[p];
+          sum += intercept[p];
```

Comment:
Thanks @anmolshkl, do you think it would be helpful to note the difference between libsvm and out API in comment somewhere, just for future reference?

Reply:
@anmolshkl -- this sounds like a good idea; flagging it for your attention.